server/api/example/[id].get.ts

```diff
@@ -1,13 +1,13 @@
 import { defineEventHandler } from 'h3'
-import type { CompatibilityEvent } from 'h3'
+import type { H3Event } from 'h3'
 import { parseDataAs, parseParamsAs, z } from '@sidebase/nuxt-parse'
 import { Example, exampleFull } from '~/server/database/entities/Example'
 
 const paramsSchema = z.object({
   id: z.string().uuid(),
 })
 
-export default defineEventHandler(async (event: CompatibilityEvent) => {
+export default defineEventHandler(async (event: H3Event) => {
```

Comment:
let's also use `eventHandler` instead of `defineEventHandler`, as written here: https://github.com/unjs/h3/releases